tests/periph_i2c/main.c

```diff
@@ -20,270 +20,550 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <errno.h>
 
 #include "periph_conf.h"
 #include "periph/i2c.h"
 #include "shell.h"
 
-#define BUFSIZE        (128U)
+#ifndef GET_I2C_CLK_SPD
+#define GET_I2C_CLK_SPD(x)  (-1)
+#endif
+
+#ifndef I2C_ACK
+#define I2C_ACK         (0)
+#endif
+
+#define INVALID_ARGS    puts("Error: Invalid number of arguments");
+
+#define BUFSIZE         (128U)
+
+#define CONVERT_ERROR   (-32768)
 
 static int i2c_dev = -1;
 
-static i2c_speed_t map_speed[] = {
-    I2C_SPEED_LOW,
-    I2C_SPEED_NORMAL,
-    I2C_SPEED_FAST,
-    I2C_SPEED_FAST_PLUS,
-    I2C_SPEED_HIGH
-};
+static inline void print_i2c_read(i2c_t dev, uint16_t *reg, uint8_t *buf,
+    size_t len)
+{
+    printf("Success: i2c_%i read %i byte(s) ", dev, len);
```

Comment:
travis complains here, should `printf("Success: i2c_%i read %i byte(s) ", dev, (int)len);`  for instance.

Reply:
casting length from `size_t` to `int` is the easiest, I guess.

Reply:
@MrKevinWeiss can you fix this small issue before merge, because it will hit us anyway when this branch will be merged back into master.

Reply:
oops missed that, sure!